swan/values.yaml

```diff
@@ -168,6 +168,8 @@ jupyterhub:
         centos7_image: "gitlab-registry.cern.ch/swan/docker-images/systemuser:v6.0.0"
         accpy_image: "gitlab-registry.cern.ch/rhenriqu/test_image_registry/swan-acc:latest"
       SpawnHandlersConfigs:
+        tn_enabled: false
+        ats_role: "swan-ats"
```

Comment:
This needs to go in the swan-cern chart, since these are settings specific to CERN.